docs/sources/variables/templates-and-variables.md

```diff
@@ -43,9 +43,9 @@ Panel titles and metric queries can refer to variables using two different synta
 - `$varname`
   This syntax is easy to read, but it does not allow you to use a variable in the middle of a word.
   **Example:** apps.frontend.$server.requests.count
-- `[[varname]]` 
-  **Note:** This syntax might be deprecated in the future.
-  Example: apps.frontend.[[server]].requests.count
+- `${var_name}` This syntax is needed when you want to interpolate a variable in the middle of an expression. 
+- `${var_name:<format>}` Gives you more control over how values are interpolated. Read [Advanced variable format options]({{< relref "advanced-variable-format-options.md" >}}) for more detail on all the formatting types.
+- `[[varname]]` Deprecated old syntax, will be removed in a future release. 
```

Comment:
```suggestion
- `[[varname]]` Do not use. Deprecated old syntax, will be removed in a future release. 
```

Reply:
I'm told this syntax is necessary for certain use cases. @daniellee , can you confirm?

Reply:
no it's not, all those use cases is covered by  ${variable}

Reply:
The issue is that there are thousands of dashboards using this the [[ ]] syntax so I don't think we will ever remove it. Or what do you say @torkelo ?

Reply:
probably not :) maybe if we have a migration for it, but that is a bit tricky